src/workloads/execution/lookup/LookupNLJ.yml

```diff
@@ -0,0 +1,345 @@
+- OperationMetricsName: Foreign1k_Scalar_Object2
+  OperationName: RunCommand
+  OperationCommand: *NLJLookupForeign1kScalarObj2Cmd
+- OperationMetricsName: Foreign1k_Scalar_Object5
+  OperationName: RunCommand
+  OperationCommand: *NLJLookupForeign1kScalarObj5Cmd
+- OperationMetricsName: Foreign1k_Object5_Scalar
+  OperationName: RunCommand
+  OperationCommand: *NLJLookupForeign1kObj5ScalarCmd
+- OperationMetricsName: Foreign1k_Scalar_Array1
+  OperationName: RunCommand
+  OperationCommand: *NLJLookupForeign1kScalarArr1Cmd
+- OperationMetricsName: Foreign1k_Scalar_Array8
+  OperationName: RunCommand
+  OperationCommand: *NLJLookupForeign1kScalarArr8Cmd
+- OperationMetricsName: Foreign1k_Array8_Scalar
+  OperationName: RunCommand
+  OperationCommand: *NLJLookupForeign1kArr8ScalarCmd
+- OperationMetricsName: Foreign10k_Scalar_Scalar
+  OperationName: RunCommand
+  OperationCommand: *NLJLookupForeign10kScalarScalarCmd
+- OperationMetricsName: Foreign10k_Scalar_Object2
+  OperationName: RunCommand
+  OperationCommand: *NLJLookupForeign10kScalarObj2Cmd
+- OperationMetricsName: Foreign10k_Scalar_Object5
+  OperationName: RunCommand
+  OperationCommand: *NLJLookupForeign10kScalarObj5Cmd
+- OperationMetricsName: Foreign10k_Object5_Scalar
+  OperationName: RunCommand
+  OperationCommand: *NLJLookupForeign10kObj5ScalarCmd
+- OperationMetricsName: Foreign10k_Scalar_Array1
+  OperationName: RunCommand
+  OperationCommand: *NLJLookupForeign10kScalarArr1Cmd
+- OperationMetricsName: Foreign10k_Scalar_Array8
+  OperationName: RunCommand
+  OperationCommand: *NLJLookupForeign10kScalarArr8Cmd
+- OperationMetricsName: Foreign10k_Array8_Scalar
+  OperationName: RunCommand
+  OperationCommand: *NLJLookupForeign10kArr8ScalarCmd
+
+# Defines how many times each test operation is executed.
+TestRepeatCount: &TestRepeatCount 10
+
+Actors:
+# The 'Setup' actor drops the database and runs in the 0th phase.
+- Name: Setup
+  Type: RunCommand
+  Database: *db
+  Thread: 1
+  Phases:
+  - Repeat: 1
+    Threads: 1
+    Database: *db
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand: {dropDatabase: 1}
+  - Phase: 1..3
+    Nop: true
+
+# The 'LoadForeign1K' / 'LoadForeign10K' / 'LoadLocalDoc100k' actors run in the 1st phase.
+- Name: LoadForeign1K
+  Type: CrudActor
+  Database: *db
+  Threads: 1
+  Phases:
+  - &Nop {Nop: true}
+  - Repeat: 1000
+    Threads: 1
+    Collection: *Foreign1K
+    Operations:
+    - OperationName: insertOne
+      OperationCommand:
+        Document: *LoadDoc
+  - *Nop
+  - *Nop
+
+- Name: LoadForeign10K
+  Type: CrudActor
+  Database: *db
+  Threads: 1
+  Phases:
+  - *Nop
+  - Repeat: 10000
+    Threads: 1
+    Collection: *Foreign10K
+    Operations:
+    - OperationName: insertOne
+      OperationCommand:
+        Document: *LoadDoc
+  - *Nop
+  - *Nop
+
+- Name: LoadLocalDoc100k
+  Type: CrudActor
+  Database: *db
+  Threads: 10
+  Phases:
+  - *Nop
+  - Repeat: 10000
```

Comment:
should this be 100k? or is this meant to be 10k?

Reply:
There're 10 thread for this actor, each thread will generate 10k docs.